pyproject.toml

```diff
@@ -47,6 +48,10 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
+extra = [
```

Comment:
is `extra` a good name? Should we name it kwcoco if that is what it contains?

Reply:
kwcoco makes sense since this is what it contains as you said